tests/conftest.py

```diff
@@ -383,3 +384,15 @@ def environment_checker(request):
 
     request.addfinalizer(get_status_after_execution)
     get_status_before_execution()
+
+
+@pytest.fixture(scope="session")
+def log_cli_level(pytestconfig):
+    """
+    Retrieves the log_cli_level set in pytest.ini
+
+    Returns:
+        str: log_cli_level set in pytest.ini or DEBUG if not set
+
+    """
+    return pytestconfig.getini('log_cli_level') or 'DEBUG'
```

Comment:
can you test if it respects the cli option that pytest provides?

Reply:
@vasukulkarni yes I did, and it does